src/store.rs

```diff
@@ -368,6 +368,19 @@ macro_rules! store { (
             }
         }
 
+        pub fn activate(&self, ifs: &'static littlefs2::fs::Filesystem<$Ifs>,
+				efs: &'static littlefs2::fs::Filesystem<$Efs>,
+				vfs: &'static littlefs2::fs::Filesystem<$Vfs>) {
+            let store_ifs = $crate::store::Fs::new(ifs);
+            let store_efs = $crate::store::Fs::new(efs);
+            let store_vfs = $crate::store::Fs::new(vfs);
+            unsafe {
+		Self::ifs_ptr().write(store_ifs);
+		Self::efs_ptr().write(store_efs);
+		Self::vfs_ptr().write(store_vfs);
```

Comment:
This looks like an invalid block indentation. Shall we introduce pre-commit checks?

Reply:
fixed by replacing tabs with spaces, although this is actually something I would like `rust-fmt` to do in the future, currently running `rust-fmt --check` will lead to lots of changes - so I would suggest to leave this (and therefore `rust-fmt` enforcement) to be done soon(tm)